core/src/main/scala/org/apache/spark/NewAccumulator.scala

```diff
@@ -199,7 +199,8 @@ private[spark] object AccumulatorContext {
    * TODO: Don't use a global map; these should be tied to a SparkContext (SPARK-13051).
    */
   @GuardedBy("AccumulatorContext")
```

Comment:
I think this annotation is no longer needed.
But first, what's the advantage of this change overall?